GoogleForCreators/web-stories-wp
# Update Moveable to 0.27.6

## Summary
Updates Moveable to 0.27.6


## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions
1. Add an image
2. Verify that rotating, dragging, and resizing the image works as expected.



### QA
See the testing instructions above.





This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #6028
